sdk/search/search-documents/swagger/Data.md

```diff
@@ -55,28 +58,5 @@ directive:
   - from: swagger-document
     where: $.definitions.IndexAction
     transform: >
-      $.properties['@search.action']['x-ms-client-name'] = '__actionType';
       $.required = ['@search.action'];
 ```
-
-
-### Change text to _text in SuggestResult
-
-```yaml
-directive:
-  - from: swagger-document
-    where: $.definitions.SuggestResult.properties['@search.text']
-    transform: >
-      $['x-ms-client-name'] = '_text'
-```
-
-### Change score to _score & highlights to _highlights in SuggestResult
-
-```yaml
-directive:
```

Comment:
If we delete this, how are we disambiguating the values returned from the server from ones on the document itself? `generatedSearchResultToPublicSearchResult` seems to still expect these properties to exist so we can assign them to the result shape

Reply:
Brought it back.